src/app/submission/pages/submission.component.ts

```diff
@@ -100,7 +100,7 @@ export class SubmissionComponent implements OnInit, OnDestroy {
 
   connectSubmissionEnvelope() {
     this.submissionDataSource = new SimpleDataSource<SubmissionEnvelope>(this.submissionEnvelopeEndpoint.bind(this));
-    this.submissionDataSource.connect(true, 15000).subscribe(submissionEnvelope => {
+    this.submissionDataSource.connect(true, 5000).subscribe(submissionEnvelope => {
```

Comment:
Could we name this magic number, please? Tx

Reply:
is this the first 5000 submissionEnvelopes?

Reply:
Actually, we don't even need this because 5000 is the default, i'll remove it. And @MightyAx it is the interval between polling in ms